docs/Components.md

```diff
@@ -0,0 +1,67 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Components
+
+## Components of Basic Calico Container Deployments
+
+All basic Calico deployments require the following components:
+
+ - **`calico/node`**: Docker image that runs the underlying calico processes (see 
+   ***Anatomy of calico/node*** below). The `calico/node` image must be deployed 
```

Comment:
You can link to section headings in a GitHub-rendered Markdown document, e.g.:

    [Anatomy of calico/node](#anatomy-of-caliconode)

Possibly not so useful when the heading is so close, but handy in general.